<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>distinta_riba</class>
 <widget class="QDialog" name="distinta_riba">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>230</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>230</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>230</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>230</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Visualizzazione</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="12" column="1">
       <widget class="QLineEdit" name="cerca_txt"/>
      </item>
      <item row="14" column="0">
       <widget class="QPushButton" name="find_txt">
        <property name="text">
         <string>Cerca</string>
        </property>
        <property name="icon">
         <iconset resource="../fabaria.qrc">
          <normaloff>:/images/edit-find.png</normaloff>:/images/edit-find.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QRadioButton" name="label_prec">
        <property name="text">
         <string>Visualizza mese precendente</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QComboBox" name="anno_sel"/>
      </item>
      <item row="4" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>11</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QRadioButton" name="label_succ">
        <property name="text">
         <string>Visualizza mese successivo</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>33</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QComboBox" name="cli_forn">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="13" column="0" colspan="2">
       <widget class="QComboBox" name="combo_find">
        <item>
         <property name="text">
          <string>Cerca per cliente o fornitore</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cerca per tipo documento</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="sel_anno">
        <property name="text">
         <string>Seleziona anno</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Cerca</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Seleziona cliente o fornitore</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>111</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="16" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="3">
         <widget class="QPushButton" name="stampa">
          <property name="text">
           <string>Stampa</string>
          </property>
          <property name="icon">
           <iconset resource="../fabaria.qrc">
            <normaloff>:/images/document-print.png</normaloff>:/images/document-print.png</iconset>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="ant_stampa">
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Anteprima</string>
          </property>
          <property name="icon">
           <iconset resource="../fabaria.qrc">
            <normaloff>:/images/document-preview.png</normaloff>:/images/document-preview.png</iconset>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="chiudi">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Chiudi</string>
          </property>
          <property name="icon">
           <iconset resource="../../backup_restore_db/images.qrc">
            <normaloff>:/images/dialog-close.png</normaloff>:/images/dialog-close.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="salda">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Mod. 
selezionati</string>
          </property>
          <property name="icon">
           <iconset resource="../fabaria.qrc">
            <normaloff>:/images/cash2.png</normaloff>:/images/cash2.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="15" column="0">
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="QComboBox" name="sel_cli_forn"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QRadioButton" name="vis_tutti">
        <property name="text">
         <string>Visualizza tutti</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTableView" name="tableView">
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>15</number>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>vis_tutti</tabstop>
  <tabstop>label_prec</tabstop>
  <tabstop>label_succ</tabstop>
  <tabstop>anno_sel</tabstop>
  <tabstop>sel_cli_forn</tabstop>
  <tabstop>cli_forn</tabstop>
  <tabstop>cerca_txt</tabstop>
  <tabstop>combo_find</tabstop>
  <tabstop>find_txt</tabstop>
  <tabstop>salda</tabstop>
  <tabstop>stampa</tabstop>
  <tabstop>ant_stampa</tabstop>
  <tabstop>chiudi</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources>
  <include location="../../backup_restore_db/images.qrc"/>
  <include location="../fabaria.qrc"/>
 </resources>
 <connections/>
</ui>
